app/javascript/dashboard/components/ChatList.vue

```diff
@@ -410,8 +410,10 @@ export default {
     },
     conversationListPagination() {
       const conversationsPerPage = 25;
+      const hasChatsOnView =
+        Array.isArray(this.chatsOnView) && this.chatsOnView.length !== 0;
```

Comment:
@nithindavid I tried with these changes, and it's working now. Could you take a look?
```suggestion
      const hasChatsOnView =
        this.chatsOnView &&
        Array.isArray(this.chatsOnView) &&
        !this.chatsOnView.length;
```